prov/verbs/src/fi_verbs.h

```diff
@@ -944,23 +964,74 @@ static inline int fi_ibv_poll_reap_unsig_cq(struct fi_ibv_ep *ep)
 	return FI_SUCCESS;
 }
 
+static inline int fi_ibv_post_send(struct fi_ibv_ep *ep, struct ibv_send_wr *wr,
+				   struct ibv_send_wr **bad_wr)
+{
+	return ibv_post_send(ep->ibv_qp, wr, bad_wr);
+}
```

Comment:
nit: the static inlines for any of the handlers won't help because these are all accessed via function pointers.